src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const styleArray = sourceString.split(';');
+
+  styleArray.forEach((style) => {
+    const [property, value] = style.split(':').map((part) => part.trim());
```

Comment:
The use of 'map' here is unnecessary since you're only trimming the parts. You can directly trim the 'property' and 'value' after splitting without mapping over the parts.